content/games.html

```diff
@@ -104,16 +104,20 @@
         var row = $('<tr class="'+rowClass+'">');
         row.append('<td>'+game['name'].replace(/\\\'/g, "'")+'</span></td>');
 
-        if (game['state'] == '1')
-          row.append('<td>'+game['location']+'<br /><span class="gameSubline"><a href="openra://'+game['address']+'">'+game['address']+'</a></span></td>');
-        else
+        if (game['state'] == '1') {
+          var handler = 'openra-' + modver[0] + '-' + modver[1] + '://';
+          if (modver[1] == 'release-20170527')
+            handler = 'openra://';
+
+          row.append('<td>'+game['location']+'<br /><span class="gameSubline"><a href="'+handler+game['address']+'">'+game['address']+'</a></span></td>');
+        } else
```

Comment:
Not gonna block this, just a suggestion: would it be better to make the server name the link, and not the address? I have a feeling that many people miss that some IPs are links.